src/app/functions/server/updateDataBrokerScanProfile.test.ts

```diff
@@ -0,0 +1,166 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { it, expect, jest } from "@jest/globals";
+import { OnerepProfileRow, UpdateableProfileDetails } from "knex/types/tables";
+
+const currentProfileDetails: OnerepProfileRow = {
+  id: 1234,
+  onerep_profile_id: 5678,
+  first_name: "First01",
+  last_name: "last01",
+  middle_name: "",
+  first_names: [],
+  middle_names: [],
+  last_names: [],
+  city_name: "City01",
+  state_code: "NY",
+  addresses: [{ city: "City01", state: "NY" }],
+  phone_numbers: ["8005553534"],
+  date_of_birth: new Date(),
+  created_at: new Date(),
+  updated_at: new Date(),
+};
+
+const newProfileDetails: UpdateableProfileDetails = {
+  first_name: "First01",
+  last_name: "last01",
+  middle_name: "",
+  first_names: [],
+  middle_names: [],
+  last_names: [],
+  addresses: [{ city: "City01", state: "NY" }],
+  phone_numbers: ["8005553534"],
+};
+
+jest.mock("./onerep", () => ({
+  updateProfile: jest.fn(),
+}));
+
+jest.mock("./refreshStoredScanResults");
+
+jest.mock("./logging", () => {
+  return {
+    logger: {
+      info: jest.fn(),
+    },
+  };
+});
+
+beforeEach(() => {
+  jest.spyOn(console, "info").mockImplementation(() => undefined);
+});
```

Comment:
Question: Why is this necessary? I don't see (nor expect) any calls to `console.info`.

Reply:
Good catch: The log that needed the mock implementation has been removed since.